include/ofi_mr.h

```diff
@@ -264,13 +264,17 @@ int ofi_mr_map_verify(struct ofi_mr_map *map, uintptr_t *io_addr,
  * registration.
  */
 
+/* libfabric internal flags starting from 60 to 63 */
+#define FI_HMEM_DATA_GDRCOPY_HANDLE	(1ULL << 60)
```

Comment:
To my understanding `ofi_mr.h` is more suited for this declaration than `ofi_hmem.h`, since this flag should be used next to `struct ofi_mr`. Please let me know otherwise.

Reply:
Ack. Will address.

Before that I would like to clarify the change to `struct fi_mr_attr`. Please see the other response.